### ISSUE TITLE
Email notification test triggers SQL error

### ISSUE THREAD

**Issue by michaelroper** [verbb/formie#459]
**Description**
When creating the email notification template, when I pressed the "Send Test Email" button on the Preview tab, I got the following error. 

![image](https://user-images.githubusercontent.com/60287/117133018-452b3c00-add6-11eb-9d03-0d48e165e070.png)

From what I can tell, the email did actually send, as I received it in my inbox, but the error was because it couldn't insert the data into the `formie_sentnotifications` table because the subject may be been too long? My subject had a few variables in it, that probably weren't expecting to have sentences of test data put through it. Maybe I'm an edge case, but FYI anyway!

**Additional info**
- Plugin version: 1.4.3
- Craft version: 3.6.13
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by michaelroper** [comment]
Sweet, thanks Josh!

**Comment by engram-design** [comment]
Fixed in [1.4.4](https://github.com/verbb/formie/releases/tag/1.4.4)
